CHANGELOG.md

```diff
@@ -14,3 +14,7 @@ All Notable changes to `Open in Git host` will be documented in this file
 ## 1.5.6 - 2016-09-26
 
 - Ability to open a specific commit from the VCS log tool window.
+
+## 1.5.7 - 2016-12-04
+
+-  Fixed: "Open in... Git Host" action works also from editor (It uses the line number from the current editor) (#24 PR by markiewb)
```

Comment:
Could this be updated to something like Fixed: Select target action now uses the correct line number when used from the editor.